src/sentry/services/hybrid_cloud/notifications/impl.py

```diff
@@ -178,20 +177,17 @@ def serialize_many(
         filter: NotificationSettingFilterArgs,
         as_user: Optional[RpcUser] = None,
         auth_context: Optional[AuthenticationContext] = None,
-        serializer: Optional[Serializer] = None,
     ) -> List[OpaqueSerializedResponse]:
-        return self._FQ.serialize_many(filter, as_user, auth_context, serializer)
+        return self._FQ.serialize_many(filter, as_user, auth_context)
 
     class _NotificationSettingsQuery(
         FilterQueryDatabaseImpl[
             NotificationSetting, NotificationSettingFilterArgs, RpcNotificationSetting, None
         ],
     ):
         def apply_filters(
-            self,
-            query: BaseQuerySet,
-            filters: NotificationSettingFilterArgs,
-        ) -> List[User]:
+            self, query: QuerySet, filters: NotificationSettingFilterArgs
+        ) -> QuerySet:
```

Comment:
`QuerySet[User]` I believe?

Reply:
`QuerySet[NotificationSetting]` actually. I've changed the superclass's annotation to `QuerySet[BASE_MODEL]`.